src/BuildLoop.cpp

```diff
@@ -25,11 +25,12 @@
 #include "RunAction.hpp"
 #include "BuildQueue.hpp"
 #include "Driver.hpp"
+#include "LeafInputSignature.hpp"
+#include "CacheClient.hpp"
 #include <stdarg.h>
 #include <algorithm>
 #include <stdio.h>
-
-
+#include <time.h>
```

Comment:
I don't think we use anything from time.h in this file?

Reply:
Fixed